### PR TITLE
data: nfa1

### PR DESCRIPTION
**Pull request by nfa1**
## Changes proposed

Added my name, bio and 2 social media links (Twitter & GitHub) to this new file.

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5873"></a>

### CODE REVIEW

File: data/nfa1.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+	"bio": "Software Engineering|Data Science|Python|SQL|R|HTML|CSS|JavaScript",
+	"links": [{
+			"name": "Follow me on Twitter",
+			"url": "https://twitter.com/nilofalvarado",
+			"icon": "FaTwitter"
+		},
+		{
+			"name": "Follow me on GitHub",
+			"url": "https://www.gitub.com/nfa1/",
+			"icon": "FaGitHub"
```
Comment (SandeepKrSuman) [inline data/nfa1.json:13]:
```diff
-  "icon": "FaGitHub"

+ "icon": "FaGithub"
```

File: data/nfa1.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+	"name": "Nilo Alvarado",
+	"type": "personal",
+	"bio": "Software Engineering|Data Science|Python|SQL|R|HTML|CSS|JavaScript",
```
Comment (SandeepKrSuman) [inline data/nfa1.json:4]: You may add proper spacing if that will look good.

> "Software Engineering | Data Science | Python | SQL | R | HTML | CSS | JavaScript"

**Review by SandeepKrSuman — CHANGES_REQUESTED** [review]

**Comment by nfa1** [comment]
Applied changes thanks!

**Review by SandeepKrSuman — APPROVED** [review]
Thanks, @nfa1 🎉 Looks good to me!

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**